virtcontainers/agent.go

```diff
@@ -155,6 +155,9 @@ type agent interface {
 	// createSandbox will tell the agent to perform necessary setup for a Sandbox.
 	createSandbox(sandbox *Sandbox) error
 
+	// cleanupSandbox will tell the agent to cleanup resources related to the Sandbox.
+	cleanupSandbox(sandbox *Sandbox) error
```

Comment:
Since you're returning `nil` from all the implementations, which makes sense BTW, I would suggest this function not to return anything (void basically).

Reply:
no `error` returns from `cleanupSandbox()` now~